library/nginx

```diff
@@ -3,24 +3,24 @@
 Maintainers: NGINX Docker Maintainers <docker-maint@nginx.com> (@nginxinc)
 GitRepo: https://github.com/nginxinc/docker-nginx.git
 
-Tags: 1.15.10, mainline, 1, 1.15, latest
+Tags: 1.15.11, mainline, 1, 1.15, latest
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 97b65112180e0c7764465aa47a974fc7af3c99ae
+GitCommit: 7d7c67f2eaa6b2b32c718ba9d93f152870513c7c
```

Comment:
I don't think this commit is right: https://github.com/nginxinc/docker-nginx/commit/7d7c67f2eaa6b2b32c718ba9d93f152870513c7c :confused:

Reply:
Ugh, I like how I've actually went through all the checklist, except actual git push.  Fixed now, sorry about the troubles.